CMakeLists.txt

```diff
@@ -89,9 +89,15 @@ else ()
   #   interfaces, etc.
   # disable -Wpointer-arith: this is a GCC extension, and doesn't work in MSVC.
   add_definitions(
-    -Wall -Wextra -Werror -Wno-unused-parameter -Wpointer-arith -g
+    -Wall -Wextra -Werror -Wno-unused-parameter -Wpointer-arith -g -std=c++11
```

Comment:
Yeah, I suppose so. Thought it looks like Travis is also at gcc 4.8.4 and clang 3.5 by default, so we'll have to update those too. Anyway, probably worth moving this convo to an actual issue. :)